fecgov/openFEC
# fix mv node in flow file

## Summary (required)

- Resolves # 3308

We found an error message during materialized views refreshing time.  
Error refreshing node %s: not found.

Comparison of all existing materialized views in the database and the message from Kabana log shows all existing materialized views in the database are refreshed.

flow.py file need to be updated to clean up the missing node.

## How to test the changes locally

Pull the branch to local and run "invoke create_sample_db".  The error message should NOT appeared during the materialized view refreshment step (the last step)

## Impacted areas of the application
List general components of the application that this PR will affect:

None

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
